packages/dev-server-api/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@react-native-community/dev-server-api",
```

Comment:
can we rename this to `@react-native-community/cli-server-api`? It's important to have a `@react-native-community/cli-` prefix, so we don't pollute the organization

Reply:
Also the directory change to `cli-server-api` would be cool

Reply:
Sounds good to me!